app/src/test/java/uk/gov/onelogin/login/usecase/SaveTokensTest.kt

```diff
@@ -0,0 +1,127 @@
+package uk.gov.onelogin.login.usecase
+
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.test.runTest
+import org.junit.jupiter.api.Test
+import org.mockito.kotlin.any
+import org.mockito.kotlin.eq
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.times
+import org.mockito.kotlin.verify
+import org.mockito.kotlin.verifyNoInteractions
+import org.mockito.kotlin.whenever
+import uk.gov.android.authentication.TokenResponse
+import uk.gov.onelogin.repositiories.TokenRepository
+import uk.gov.onelogin.tokens.Keys
+import uk.gov.onelogin.tokens.usecases.GetPersistentId
+import uk.gov.onelogin.tokens.usecases.SaveToOpenSecureStore
+import uk.gov.onelogin.tokens.usecases.SaveToSecureStore
+
+class SaveTokensTest {
+    private val saveTokens by lazy {
```

Comment:
I would say that this should be created fresh for each test, even if it doesn't have any information persistence, just because the test shouldn't have to know the internals of how the class is set up. 
This is being a bit picky though